tests/test_craft_comment_with_dependency_diff.bats

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bats
+
+setup() {
```

Comment:
This test is not executed on CI - should we update the config?

Reply:
`tests/*.bats` tests are executed as part of the `make buildkite-plugin-test` command (which is a Make target itself called as part of `make test`). So [they are properly executed on CI](https://buildkite.com/automattic/ci-toolkit-buildkite-plugin/builds/1081#01952877-170c-4a31-944a-7411258770c6/127-154), it's just a bit more hidden that you probably hoped 😅 .

Reply:
I'm blind, but this is a great news :D thanks!